app/components/Layout.tsx

```diff
@@ -33,9 +40,12 @@ type Props = {
 };
 
 const Layout: React.FC<Props> = ({ children, session, title }) => {
+  const enableTimeMachine = runtimeConfig.ENABLE_MOCK_TIME;
+  const environment = runtimeConfig.OPENSHIFT_APP_NAMESPACE;
+  const isTestEnv = environment ? !environment.endsWith('-prod') : false;
```

Comment:
```suggestion
```

We don't need to check the environment, we're already doing that in `config.js`. This will also only display the time machine if there is an `OPENSHIFT_APP_NAMESPACE`, which disables the local file upload: we may want both these features enabled at the same time